cloud_governance/common/mails/templates/cro_monitor_budget_remain_alert.j2

```diff
@@ -2,11 +2,20 @@
     <p>Hi {{name}},</p>
 </div>
 <div>
-    <p>You completed $ {{used_budget}} / $ {{budget}} of budget till today of the TicketId: {{ticket_id}}.
-    Remaining budget will be $ {{remain_budget}}.</p>
-    <p>Open the budget extension if further needed.</p>
-    <p>This ticket will be closed, if the budget will exceeds 110%.</p>
-    <p>You can extend the budget <a href="{{portal}}/extend/budget">here</a>.</p>
-    <p>Visit the <a href="{self.__portal}/wiki/clouds">wiki page</a> to get more information</p>
+    <p>Summary of you budget request TicketId: {{ ticket_id }}</p>
+    <ul>
+        <li>Allocated Budget: <b>$ {{ budget }}</b></li>
+        <li>Cost as of today: <b>$ {{ used_budget }}</b></li>
+        <li>Remaining Budget: <b>$ {{ remain_budget }}</b></li>
```

Comment:
can you remove the space after the `$` sign? (for proper formating of the currency)
```suggestion
        <li>Allocated Budget: <b>${{ budget }}</b></li>
        <li>Cost as of today: <b>${{ used_budget }}</b></li>
        <li>Remaining Budget: <b>${{ remain_budget }}</b></li>
```